client/src/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=0">
```

Comment:
Thanks !

`user-scalable=0` does not work on iOS, only Javascript can prevent scaling :  https://stackoverflow.com/questions/37808180/disable-viewport-zooming-ios-10-safari

It's working on Android but I suggest we only remove pinch and zoom for the mobile mode and not for the desktop mode, otherwise we may break the touch experience on some devices (especially phones that needs zoom sometimes when we uses the desktop mode).